src/envo/envo-edit.owl

```diff
@@ -44974,6 +44975,17 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (stream water)
+
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03605006> "ORCID:0000-0002-4366-3088")
```

Comment:
I think we have an inconsistency here: we're using Dublin Core for this and creator date for broader interoperability. The ORCID should also be an IRI and the full URL for easier traversal

Reply:
We should all have Protege set up in exactly the same way and this should be populated automatically